src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -48,6 +48,7 @@ const Html = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
             left: 0,
             width,
             height,
+            pointerEvents: 'none', // let pointer events pass through to canvas
```

Comment:
Fix for the only regression caused by the React v17 upgrade. Now, `ReactDOM` registers its event handlers on the root element passed to `ReactDOM.render` instead of on the body. This means that our `Html` components now catch all the DOM events and stop their propagations. Since we want the `canvas` element to receive the events and yet still be positioned underneath the `Html` elements (so the SVG tooltip and grid remain visible), we have to make the `Html` elements "invisible" to DOM events with `pointer-events: none`.